### DISCUSSION TITLE
YQ to validate or lint a directory of yaml files

### DISCUSSION THREAD

**Discussion by robertgates55** [mikefarah/yq#744] [Q&A]
I'd like to use yq (4) to perform a basic validation of a directory tree containing a large number of yaml files (~20k) in one go.

I've piped a find command into yq, but the performance overhead of running so many sequential yq eval commands is too high.

Is there any way to operate on a directory or a glob pattern?

**Comment by mikefarah** [comment]
I'm not sure what you are doing, I'd just use find + xargs - is there an issue with that?
